### DISCUSSION TITLE
Satranç turnuvası sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#67] [Zeka Sorusu] [answered]
Ayşegül, Beyza, Cagdas , Deniz ve Elisa adlı oyuncular tek turlu bir satranç turnuvası yaptılar. (her biri diğeri ile sadece 1 kez oynadı)
Turnuva sonunda aldıkları puanlara göre birinciden başlayarak (A, B, C, D, E ) şeklinde sıralandılar.
Beraberliklere yarım galibiyetlere 1 puan verilen turnuvanın ardından B ve E şöyle dedi.
B: Turnuvayı hiç mağlup olmadan bitiren tek oyuncu benim.
E: Turnuvayı hiç galibiyet almadan bitiren tek oyuncu benim.
Bu verilerle skor tablosunu oluşturabilir misiniz?

**Comment by FatihOnder81** [comment, marked as answer]
Bir kişi en fazla 4 puan alabilir ama 2.oyuncu hiç mağlup olmadan bitiren tek oyuncu olduğuna göre 1. oyuncuda illaki 1 kere mağlup olması gerekir maçlar şu şekildedir ab ac ad ae bc bd be cd ce de harfler ,s,mlerin kısaltılmasıdır toplamda 10 maç olduğu için toplam 10 puan olması gerekir o yüzden puan tablosu 3 2.5 2 1.5 1 şeklindedir
